openjdk/mmtkMutator.cpp

```diff
@@ -6,6 +6,10 @@
 size_t MMTkMutatorContext::max_non_los_default_alloc_bytes = 0;
 
 MMTkMutatorContext MMTkMutatorContext::bind(::Thread* current) {
+    if (FREE_LIST_ALLOCATOR_SIZE != sizeof(FreeListAllocator)) {
```

Comment:
This line is not properly indented.
```suggestion
  if (FREE_LIST_ALLOCATOR_SIZE != sizeof(FreeListAllocator)) {
```